### ISSUE TITLE
Currently no way to utilize stacked tiles with tileColorize

### ISSUE THREAD

**Issue by Blindmikey** [ondras/rot.js#136]
There doesn't seem to be a way to utilize both great features. Stacked tiles in ROT.Display.draw() works only when tileColorize is not set. 

Proposed syntax would be 
```
ROT.Display.draw( x, y, ['#', '@'], ['fgcolor1', 'fgcolor2'], ['bgcolor1', 'bgcolor2'] );
```

**Comment by ondras** [comment]
Makes sense! Also, this looks like a pretty straightforward improvement.

**Comment by ondras** [comment]
@Blindmikey I believe I fixed this (and added a small demo to the interactive manual) in https://github.com/ondras/rot.js/commit/5375817b392b6276bbe0f38c15dd555da5d5ebf2

**Comment by Blindmikey** [comment]
This is amazing, thank you so much!
